content/tutorial/tutorial.md

```diff
@@ -12,185 +12,190 @@ redirect_from:
   - "docs/tutorial-zh-CN.html"
 ---
 
-This tutorial doesn't assume any existing React knowledge.
+यह ट्यूटोरियल किसी भी मौजूदा React ज्ञान को ग्रहण नहीं करेगा।
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## ट्यूटोरियल शुरू करने से पहले {#before-we-start-the-tutorial}
 
-We will build a small game during this tutorial. **You might be tempted to skip it because you're not building games -- but give it a chance.** The techniques you'll learn in the tutorial are fundamental to building any React apps, and mastering it will give you a deep understanding of React.
+हम इस ट्यूटोरियल के दौरान एक छोटा Game बनाएंगे। **आप इसे छोड़ सकते हैं क्योंकि आप Game नहीं बना रहे हैं -- लेकिन इसे एक मौका दें।** ट्यूटोरियल में आपके द्वारा सीखी जाने वाली तकनीकें किसी भी React ऐप्स के निर्माण के लिए मौलिक हैं, और इसमें महारत हासिल करने से आपको React की गहरी समझ मिलेगी।
```

Comment:
any reason for not translating game?

Reply:
आप इसे छोड़ सकते हैं क्योंकि आप Game नहीं बना रहे हैं -- लेकिन इसे एक मौका दें। -> does not seem to be the correct translation for you might be tempted to skip

Reply:
sounds good